src/params.c

```diff
@@ -110,23 +141,10 @@ void MAMACALLTYPE  zmqBridgeMamaTransportImpl_parseNamingParams(zmqTransportBrid
    // the name of the transport
    const char* name = impl->mName;
 
-   impl->mNamingWaitForConnect = getInt(name, "naming.wait_for_connect", 1);
-   impl->mNamingConnectInterval = getFloat(name, "naming.connect_interval", .1) * ONE_MILLION;    // micros
-   impl->mNamingConnectRetries = getInt(name, "naming.connect_retries", 100);
-   impl->mNamingReconnect = getInt(name, "naming.retry_connects", 1);
-   impl->mNamingReconnectInterval = getFloat(name, "naming.retry_interval", 10) * 1000;           // millis
-   double f = getFloat(name, "naming.beacon_interval", 1);
-   if (f <= 0) {
-      impl->mBeaconInterval = 0;
-   }
-   else {
-      wInterlocked_set(f * 1000, &impl->mBeaconInterval);                                         // millis
-      if (impl->mBeaconInterval < 100) {
-         // cant be less than 100 ms
-         MAMA_LOG(MAMA_LOG_LEVEL_WARN, "beacon_interval cannot be less than 100ms");
-         wInterlocked_set(100,  &impl->mBeaconInterval);
-      }
-   }
+   impl->mNamingWaitForConnect = getInt(name, "naming.wait_for_connect", 1, 0);
+   impl->mNamingConnectInterval = getFloat(name, "naming.connect_interval", .1, .1) * 1000000.0;    // micros
+   impl->mNamingConnectRetries = getInt(name, "naming.connect_retries", 100, 10);
+   impl->mBeaconInterval = getFloat(name, "naming.beacon_interval", 1, .1) * 1000.0;    // millis;
```

Comment:
In the past we were able to set beacon_interval to 0, that doesn't appear to be possible in the current code.

```
   double f = getFloat(name, "naming.beacon_interval", 1);
￼   if (f <= 0) {
￼      impl->mBeaconInterval = 0;
￼   }
```

Reply:
Good catch!  You're right, and I just committed a fix.  (I had thought you were referring to https://github.com/zeromq/libzmq/pull/3966, but that's something else).

Thanks!